### ISSUE TITLE
1.19.3 Forge port waiting on deferred registry fix from Forge

### ISSUE THREAD

**Issue by TelepathicGrunt** [TelepathicGrunt/Bumblezone#177]
issue report here: https://github.com/MinecraftForge/MinecraftForge/issues/9199#issuecomment-1345640904

I have too many registries using the deferred registry for me to even try to convert to registry events.
